app/src/main/java/org/wikipedia/main/MainActivity.kt

```diff
@@ -61,10 +63,20 @@ class MainActivity : SingleFragmentActivity<MainFragment>(), MainFragment.Callba
     }
 
     override fun onResume() {
+        maybeShowPlacesSurvey()
         super.onResume()
         invalidateOptionsMenu()
     }
 
+    private fun maybeShowPlacesSurvey() {
+        binding.root.postDelayed({
+            if (Prefs.shouldShowOneTimePlacesSurvey == PlacesFragment.SURVEY_SHOW && !isDestroyed) {
```

Comment:
Do we need to move `!isDestroyed` to the front in case the `PlacesFragment` gets destroyed?